Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -255,6 +255,27 @@ public let CodeGenerators: [CodeGenerator] = [
         b.callMethod(methodName!, on: obj, withArgs: arguments)
     },
 
+    CodeGenerator("MethodCallWithSpreadGenerator", input: .object()) { b, obj in
+        guard b.mode != .conservative else { return }
+
+        var methodName = b.type(of: obj).randomMethod()
```

Comment:
Let's simplify this here and below to just `let methodName = b.type(of: obj).randomMethod() ?? b.genMethodName()`